mattermost/mattermost-webapp
# MM-23573: Remove Segment.

#### Summary
Remove Segment.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-23573

#### QA Testing Steps

Start up a spin mint and check that basic functionality works (log in, post message, change channels). Then talk to Bizops Team to check that **client side** Rudder data is being received from your test server.
